src/main/java/world/bentobox/bentobox/BentoBox.java

```diff
@@ -291,8 +294,6 @@ public void onDisable() {
         if (islandsManager != null) {
             islandsManager.shutdown();
         }
-        // Close all async database tasks
-        shutdown = true;
     }
```

Comment:
Moving this to the top is somewhat moot because the new queue processor uses the Bukkit task scheduler which automatically cancels async repeating tasks. However, if the onDisable() is called when *not* shutting down the server, then this will immediately stop any async saving and clear the way for the islandsManager to shutdown.